app-android/src/main/java/com/ustadmobile/port/android/view/binding/ImageViewBindings.kt

```diff
@@ -110,6 +111,21 @@ fun ImageView.setImageLookupMap(imageLookupMap: Map<Int, Int>?, imageLookupFallb
     updateFromImageLookupMap()
 }
 
+@BindingAdapter(value=["iconStatusFlag"])
+fun ImageView.setIconOnStatusFlag(statusFlag: Int){
```

Comment:
This need an else branch. This when clause is used in a recyclerview. You might need to make the statusFlag nullable to make sure that when a view is recycled from an item that has a status flag to one that does not it displays correctly.